yaksh/models.py

```diff
@@ -944,7 +942,7 @@ class Question(models.Model):
     snippet = models.TextField(blank=True)
 
     # user for particular question
-    user = models.ForeignKey(User, related_name="user")
+    user = models.ForeignKey(User, related_name="created_question")
```

Comment:
I feel `created_question` as related_name is not proper maybe this should be changed to question_user or something better.

Reply:
```created_question``` is proper!
Coz we will call  ```user.created_question``` will give question created by the user.
Else we need to call  ```user.user``` which returns question created